front/src/applications/operationalStudies/Home.tsx

```diff
@@ -29,24 +30,47 @@ type SortOptions =
   | 'LastModifiedAsc'
   | 'LastModifiedDesc';
 
-export default function HomeOperationalStudies() {
+const HomeOperationalStudies = () => {
   const { t } = useTranslation('operationalStudies/home');
   const safeWord = useSelector(getUserSafeWord);
   const [sortOption, setSortOption] = useState<SortOptions>('LastModifiedDesc');
   const [filter, setFilter] = useState('');
   const [filterChips, setFilterChips] = useState('');
   const [deleteProject] = osrdEditoastApi.endpoints.deleteProjectsByProjectId.useMutation();
 
+  const [getStudies] = osrdEditoastApi.endpoints.getProjectsByProjectIdStudies.useLazyQuery();
+  const [getScenarios] =
+    osrdEditoastApi.endpoints.getProjectsByProjectIdStudiesAndStudyIdScenarios.useLazyQuery();
+
   const {
     selectedItemIds: selectedProjectIds,
     setSelectedItemIds: setSelectedProjectIds,
     items: projectsList,
     setItems: setProjectsList,
     toggleSelection: toggleProjectSelection,
     deleteItems,
-  } = useMultiSelection<ProjectWithStudies | SearchResultItemProject>((projectId) => {
+  } = useMultiSelection<ProjectWithStudies | SearchResultItemProject>(async (projectId) => {
+    // For each scenario in the selected projects, clean the local storage if a manchette is saved
+    const { data: studies } = await getStudies({ projectId });
+    if (studies) {
+      const promisedScenarios = studies.results.map(async (study) => {
+        const { data } = await getScenarios({
+          projectId,
+          studyId: study.id,
+        });
+        return data?.results;
+      });
+
+      const scenarios = await Promise.all(promisedScenarios);
+
+      scenarios.flat().forEach((scenario) => {
+        if (scenario) cleanScenarioLocalStorage(scenario.timetable_id);
+      });
+    }
```

Comment:
Do you think you can move this code in a helper ? In modules/projects/helpers ?
You can use it directly in AddOrEditProject

Reply:
Since it's just a temporary code, I was a little lazy to do that but you're right it's better in the meantime :)